aldryn_apphooks_config/tests/test_config.py

```diff
@@ -28,6 +28,8 @@ def setUp(self):
         self.language = settings.LANGUAGES[0][0]
         self.root_page = api.create_page(
             'root page', self.template, self.language, published=True)
+        if hasattr(self.root_page, 'set_as_homepage'):
+            self.root_page.set_as_homepage()
```

Comment:
this is a bit confusing, 3.4 this method doesn't exist, would the test still pass? if so, do we need this really as the homepage?

Reply:
In 3.4 the first page is automatically published and set as root. This allows to mimic the same behavior in 3.5 and have the same page tree structure (and paths) across all versions
I can add a comment to explain why it's needed

Reply:
Yes, a comment would be good.
Thanks!

Reply:
Comment added